test/client.test.ts

```diff
@@ -447,6 +447,27 @@ describe('client', async () => {
       }
     })
 
+    test.skipIf(false)('can cancel request with an abort controller signal', async () => {
```

Comment:
```suggestion
    test.skipIf(isEdge && typeof AbortController === 'undefined')('can cancel request with an abort controller signal', async () => {
```
Doing this will ensure the test isn't run if the test run is in the vercel edge runtime, as it doesn't support `nock`. And it also skips older node environments that doesn't have `AbortController` so now the tests should pass